src/Storage/Commands.Storage/Common/Cmdlet/EnableAzureStorageDeleteRetentionPolicy.cs

```diff
@@ -0,0 +1,85 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.WindowsAzure.Commands.Storage.Common.Cmdlet
+{
+    using Microsoft.WindowsAzure.Commands.Storage.Model.ResourceModel;
+    using Microsoft.WindowsAzure.Storage.Shared.Protocol;
+    using System;
+    using System.Management.Automation;
+    using System.Security.Permissions;
+
+    /// <summary>
+    /// Enable azure storage service DeleteRetentionPolicy, currently only enabled on Blob service
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Enable, StorageNouns.ServiceDeleteRetentionPolicy, SupportsShouldProcess = true),
+        OutputType(typeof(PSDeleteRetentionPolicy))]
+    public class EnableAzureStorageServiceDeleteRetentionPolicyCommand : StorageCloudBlobCmdletBase
```

Comment:
Enable-AzureStorageSoftDelete

Reply:
As discussed with Mark, the alias for the cmdlets is added: Enable-AzureStorageSoftDelete